.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # GLOBAL OWNER
-*   @ProjectPythia/infrastructure
\ No newline at end of file
+*   @ProjectPythiaTutorials/infrastructure
```

Comment:
You'll need to create a team that matches this.  You can see the red dot next to the line is GitHub's way of telling you this won't work (Unknown Owner).

Reply:
I made the team? Let me double check.

Reply:
https://github.com/orgs/ProjectPythiaTutorials/teams/infrastructure

Reply:
Might just take a second. I was staring at this trying to find my spelling error with no luck.